sdk/servicebus/service-bus/test/invalidParameters.spec.ts

```diff
@@ -43,14 +43,14 @@ describe("invalid parameters", () => {
     });
 
     it("Peek: Invalid maxMessageCount for Queue", async function(): Promise<void> {
-      const peekedResults = await receiver.diagnostics.peek(-100);
-      should.equal(peekedResults.length, 0);
+      const browsedMessages = await receiver.browseMessages({ maxMessageCount: -100 });
```

Comment:
Not that this needs to change in this PR, but I'm surprised this doesn't throw instead since you'll never get a result with maxMesasgeCount set to a negative number.

Reply:
We have code in place that treats negative numbers as 0. Am guessing that is the same approach for `receiveBatch` as well?